README.md

```diff
@@ -76,8 +76,8 @@ The following information should additionally be available in the witness:
 
 | key | Meaning | Allowed in Violation Witnesses | Allowed in Correctness Witnesses |
 | --- | --- | ---- | ---- |
-| threadId | Represents the currently active thread for the transition. If no ``threadId`` is given, any thread can be active. The value should be a unique identifer for a thread. | Yes | Yes |
-| createThread | The currently active thread (value of ``threadId``) creates a new thread (value of ``createThread``) . In general, using a ``threadId`` is only allowed after creating a matching thread. The new thread's function can be entered on a second transition following this transition, such that the transition with the ``enterFunction`` key has the ``threadId`` of the created thread. When the function of the thread is exited, the thread is assumed to be terminated and its ``threadId`` should no longer be used. | Yes | Yes |
+| threadId | ``createThread`` is used in the analysis of concurrent programs as part of the source-code guard of a transition and restricts the set of CFA edges matched by the source-code guard to operations where a new thread is created. The value of ``data`` elements with this key is an identifier for the new thread. Any string may be used as an identifier, provided that it is unique in the document in the sense that for any run through the automaton, there may not be two different threads that (1) are both active (i.e., created but not yet destroyed) and (2) share the same identifier. The ``attr.type`` attribute of this key is ``string``. | Yes | Yes |
+| createThread | ``threadId`` is used in the analysis of concurrent programs as part of the source-code guard of a transition and represents the currently active thread for the transition. The value of ``data`` elements with this key must be an identifier of an active (i.e., created but not yet destroyed) thread. If a transition has ``data`` elements where one specifies a ``threadId`` and another one uses the ``createThread`` key, the ``threadId`` refers to the thread that creates the new thread, not the created thread. The ``attr.type`` attribute of this key is ``string``. | Yes | Yes |
```

Comment:
the key name in the first column doesn't match the description in the second column, looks like they were accidentally switched.

Reply:
Indeed. Fixed.